class_layout.lua

```diff
@@ -5,7 +5,7 @@ digtron.DigtronLayout.__index = digtron.DigtronLayout
 -- Creation
 
 local get_node_image = function(pos, node)
-	local node_image = {node=node, pos={x=pos.x, y=pos.y, z=pos.z}}
+	local node_image = {node=node, pos=vector.copy(pos)}
```

Comment:
`vector.copy` requires Minetest 5.5.0: https://github.com/luanti-org/luanti/commit/2cefe51d3b9bc4f3ae18854e171a06ea83e9cb25

Please add `min_minetest_version` to `mod.conf` such that it is picked up by ContentDB automatically. Furthermore, a presence check with a pleasant `error("Your Luanti/Minetest version is too old. Please update to 5.5.0 or newer.")` user "message" upon startup would be nice to have such that there's no issues/errors during runtime.